### ISSUE TITLE
Error in prime 5.0.0 .. autocomplete -> Cannot read property 'nativeElement' of undefined

### ISSUE THREAD

**Issue by huineng** [primefaces/primeng#4483]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report
```
**Current behavior**
```
Cannot read property 'nativeElement' of undefined
```

**Minimal reproduction of the problem with instructions**

this line is producing an error
https://github.com/primefaces/primeng/blob/master/src/app/components/autocomplete/autocomplete.ts#L610

```
return (target === this.dropdownButton.nativeElement || target.parentNode === this.dropdownButton.nativeElement);
```

there's no such thing as a dropdownButton did you mean ?
```
return (target === this.el.nativeElement || target.parentNode === this.el.nativeElement);
```

thanks

**Comment by sudheerj** [comment]
The dropdownButton object is the view child of auto complete container. So it should be defined for dropdown options. I can't replicate the issue. So Can you please provide the plunker to replicate the issue.

**Comment by huineng** [comment]
well it's right here 

![image](https://user-images.githubusercontent.com/3482445/33019949-afa43b26-cdfc-11e7-8c46-280175ed6c37.png)

**Comment by mdmazhar** [comment]
same here.

![image](https://user-images.githubusercontent.com/10234464/33041658-15ef3858-ce0d-11e7-97a7-b2e40b35a174.png)

**Comment by lukasweber** [comment]
I made a little fix for this issue. 

Hope it gets approved as soon as possible.

**Comment by aamir1995** [comment]
Please merge this PR for the upcoming release.

**Comment by jizza** [comment]
+1 (#4514)

**Comment by cagataycivici** [comment]
Fix checked in, we'll do 5.0.1 later today or tomorrow morning.

**Comment by aamir1995** [comment]
Thanks @cagataycivici .
